content/fr/docs/reference/glossary/containerd.md

```diff
@@ -0,0 +1,19 @@
+---
+title: containerd
+id: containerd
+date: 2019-05-14
+full_link: https://containerd.io/docs/
+short_description: >
+  Un environnement d'exécution des conteneurs qui met l'accent sur la simplicité, la robustesse et la portabilité.
+
+
+aka:
+tags:
+- tool
+---
+ Un environnement d'exécution des conteneurs qui met l'accent sur la simplicité, la robustesse et la portabilité.
+
+<!--more-->
+
+Containerd est un environnement d'exécution de {{< glossary_tooltip text="container" term_id="container" >}}
```

Comment:
```suggestion
Containerd est un environnement d'exécution de {{< glossary_tooltip text="conteneur" term_id="container" >}}
```
?

Reply:
En effet, le `text` devrait être traduit, que le `term_id` reste inchangé.

Reply:
That totally makes sense! Thank you to both of you!